src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -6458,6 +6513,20 @@ namespace SurfaceGeometry {
                                     cCurrentModuleObject + "=\"" + state.dataSurfaceGeometry->SurfaceTmp(SurfNum).Name + "\", " +
                                         state.dataIPShortCut->cAlphaFieldNames(3) + " not found=\"" + state.dataIPShortCut->cAlphaArgs(3));
                     ErrorsFound = true;
+                } else {
+                    if (state.dataSysVars->shadingMethod == DataSystemVariables::ShadingMethod::PixelCounting) {
+                        ShowSevereError(state,
+                                        format("{}=\"{}\" has an active {}=\"{}\";",
+                                               cCurrentModuleObject,
+                                               state.dataSurfaceGeometry->SurfaceTmp(SurfNum).Name,
+                                               state.dataIPShortCut->cAlphaFieldNames(3),
+                                               state.dataIPShortCut->cAlphaArgs(3)));
+                        ShowContinueError(state, "The Shading Calculation Method of choice is \"PixelCounting\". With this choice, ");
+                        ShowContinueError(
+                            state, "the shading surfaces will be treated as completely opaque (transmittance = 0) during the shading calculation, ");
+                        ShowContinueError(state, "which may result in significant inaccurate or unexpected results.");
+                        ShowContinueError(state, "It is suggested to use another Shading Calculation Method, such as \"PolygonClipping\".");
+                    }
                 }
```

Comment:
This is the adequate location to add the warning for attached shading surfaces.

Reply:
And this else can be removed?